### DISCUSSION TITLE
Rule trigger with stat

### DISCUSSION THREAD

**Discussion by bananajoe86** [arendst/Tasmota#19346] [Q&A]
Hey Guy´s, I need your help.
I implemented a Fingerprint sensor and it works well. If a finger is detected I got a Id < 200
`stat/tasmota_finger/RESULT = {"FPrint":{"Id":254,"Confidence":254}}`
Is it now possible to start a rule with this stat? My idea: 
`Rule1 ON FPrint < 200 DO Power3 1 ENDON`

Can somebody help me?

**Comment by sfromis** [comment]
Try correct syntax..... `on FPrint#Confidence<200 do Power3 1 endon`
